website/docs/contributing.md

```diff
@@ -19,7 +19,7 @@ Please take a look at this post on [how to contribute to open source software](h
 ## Get Started
 
 You don’t have to be a developer to contribute to FerretDB projects, you can even get started by helping us improve this documentation.
-If you have any questions or suggestions on how we can improve, kindly join our [Slack chat](/intro/#community) or [GitHub discussions](https://github.com/FerretDB/FerretDB/discussions).
```

Comment:
We link to `/intro/#community` in a few blog posts and maybe in other places

Reply:
hmm, I tried to search, but it seems that I didn't find all the places... Will search again

Reply:
FWIW, a new GitHub search finds them too: https://github.com/search?q=repo%3AFerretDB%2FFerretDB%20%2Fintro%2F%23community&type=code

Reply:
No, I don't see other places.
All the blog posts are in this repo as well as all the GitHub docs.

Anyways, `/intro` will redirect to `/`, so it's not 404.

Reply:
> No, I don't see other places (apart from those four).

There were two; f5ea894560a0049200e5f55fcb441ea2869c0f2e was made after my comment

Reply:
Yes, yes, I mean, I don't see other places after these four. There was a previous version of my comment, and I updated it after making the changes in the places you pointed... Maybe it was better to leave a new comment.